src/util/searchHelpers.ts

```diff
@@ -48,15 +69,24 @@ export const frontPageSearchSuggestion = searchResult => {
     frontpageSearch: { learningResources, topicResources },
   } = searchResult;
 
+  if (!learningResources?.suggestions || !topicResources?.suggestions) {
+    return;
+  }
+
   const suggestions = learningResources.suggestions
-    .concat(topicResources.suggestions)
-    .map(s => s?.suggestions?.[0]?.options?.[0])
+    ?.concat(topicResources.suggestions)
+    .map(s => s.suggestions?.[0]?.options?.[0])
     .filter(s => !!s)
-    .sort((a, b) => b.score - a.score);
+    .sort((a, b) => b?.score! - a?.score!);
```

Comment:
Finnes det en måte man kan få typescript til å innse at ting ikke er nullable her?

Reply:
Ikke med funksjonell programmering. Typescript har ikke støtte for det enda desverre.